check_yum

```diff
@@ -268,7 +268,10 @@ class YumTester:
 		re_no_security_updates_available_rhel5 = re.compile("No packages needed, for security, \d+ available")
 		re_no_security_updates_available_rhel6 = re.compile("No packages needed for security; \d+ packages available")
 		summary_line_found = False
+		excluded_packages_found = False
 		for line in output:
+			if "excluded" in line:
```

Comment:
I know you mentioned about this falsely matching, but adding a space on either side should improve this. I have a couple other ideas that I'll mention elsewhere.